.github/workflows/python-tests.yml

```diff
@@ -0,0 +1,75 @@
+name: Test suite
+
+on:
+  push:
+  pull_request:
+  schedule:
+  - cron: 1 0 * * *  # Run daily at 0:01 UTC
+
+jobs:
+  tests:
+    name: 👷
+    runs-on: ${{ matrix.os }}
+    strategy:
+      # max-parallel: 5
+      matrix:
+        python-version:
+        - 3.7
+        - 3.6
+        - 3.5
+        - 2.7
+        os:
+        - ubuntu-18.04
+        - ubuntu-16.04
+        - macOS-10.14
```

Comment:
```suggestion
        - macOS-latest
```
https://github.blog/changelog/2019-10-31-github-actions-macos-virtual-environment-is-updating-to-catalina-and-dropping-mojave-support